lib/commands/asm/network/vpnGateway._js

```
@@ -0,0 +1,367 @@
+    } finally {
+      progress.end();
+    }
+
+    var output = this.cli.output;
+    this.cli.interaction.formatOutput(connectionList.connections, function (outputData) {
+      if (outputData || outputData.length === 0) {
+        output.warn($('No VPN Gateway connections found'));
+      } else {
+        var indent = 0;
+        output.header('Connections', indent, true);
+
+        indent += 2;
+        var counter = 0;
+        outputData.forEach(function (item) {
+          output.header(util.format($('Connection %s', counter)), indent);
+          indent += 2;
+          output.nameValue('Local network site name', item.localNetworkSiteName, indent);
+          output.nameValue('State', item.connectivityState, indent);
+          output.nameValue('Bytes of data transferred in', item.ingressBytesTransferred, indent);
+          output.nameValue('Bytes of data transferred out', item.egressBytesTransferred, indent);
+          output.nameValue('Last connection established', item.lastConnectionEstablished, indent);
+
+          if (item.allocatedIpAddresses) {
+            output.list('VPN Client IP Addresses', item.allocatedIpAddresses, indent);
+          }
+
+          if (item.lastEvent) {
+            output.nameValue('Last event ID', item.lastEvent.id, indent);
+            output.nameValue('Last event message', item.lastEvent.message, indent);
+            output.nameValue('Last event timestamp', item.lastEvent.timestamp, indent);
+          }
+          indent -= 2;
+          counter++;
+        });
+      }
+    });
+  },
+
+  get: function (networkName, _) {
+    var gateway = null;
+    var progress = this.cli.interaction.progress(util.format($('Looking up network VPN Gateway in virtual network "%s"'), networkName));
+    try {
+      gateway = this.networkManagementClient.gateways.get(networkName, _);
+    } catch (e) {
+      if (e.code === 'BadRequest') {
```

what we want to hide this error? Any specific reason, that since we this could be a bad payload